docs/using-radicle/pushing-changes.md

```diff
@@ -0,0 +1,46 @@
+---
+id: pushing-changes
+title: Pushing changes
+---
+
+To push code to Radicle, you first need to make sure your shell is configured
+correctly by following the instructions in the [Getting Started][gs] section to
+add the Radicle remote helper.
+
+After the Radicle remote helper is configured, pushing to Radicle is as simple
+as pushing to any other remote, with the Radicle network being known as 'rad'.
```

Comment:
This is not right. It should say something like, "with the remote pointing to your project being known as `rad`". Also, a good place to differentiate the monorepo and the working copy.

Reply:
I'm going to address this in a seperate issue.